crypto/idea/idea_local.h

```diff
@@ -17,71 +19,6 @@ if (ul != 0) \
-                        /* fall through */                              \
-                        case 6: l2|=((unsigned long)(*(--(c))))<<16; \
-                        /* fall through */                              \
-                        case 5: l2|=((unsigned long)(*(--(c))))<<24; \
-                        /* fall through */                              \
-                        case 4: l1 =((unsigned long)(*(--(c))))    ; \
-                        /* fall through */                              \
-                        case 3: l1|=((unsigned long)(*(--(c))))<< 8; \
-                        /* fall through */                              \
-                        case 2: l1|=((unsigned long)(*(--(c))))<<16; \
-                        /* fall through */                              \
-                        case 1: l1|=((unsigned long)(*(--(c))))<<24; \
-                                } \
-                        }
-
-/* NOTE - c is not incremented as per l2n */
-#define l2nn(l1,l2,c,n) { \
-                        c+=n; \
-                        switch (n) { \
-                        case 8: *(--(c))=(unsigned char)(((l2)    )&0xff); \
-                        /* fall through */                                    \
-                        case 7: *(--(c))=(unsigned char)(((l2)>> 8)&0xff); \
-                        /* fall through */                                    \
-                        case 6: *(--(c))=(unsigned char)(((l2)>>16)&0xff); \
-                        /* fall through */                                    \
-                        case 5: *(--(c))=(unsigned char)(((l2)>>24)&0xff); \
-                        /* fall through */                                    \
-                        case 4: *(--(c))=(unsigned char)(((l1)    )&0xff); \
-                        /* fall through */                                    \
-                        case 3: *(--(c))=(unsigned char)(((l1)>> 8)&0xff); \
-                        /* fall through */                                    \
-                        case 2: *(--(c))=(unsigned char)(((l1)>>16)&0xff); \
-                        /* fall through */                                    \
-                        case 1: *(--(c))=(unsigned char)(((l1)>>24)&0xff); \
-                                } \
-                        }
-
-#undef n2l
-#define n2l(c,l)        (l =((unsigned long)(*((c)++)))<<24L, \
-                         l|=((unsigned long)(*((c)++)))<<16L, \
-                         l|=((unsigned long)(*((c)++)))<< 8L, \
-                         l|=((unsigned long)(*((c)++))))
-
-#undef l2n
-#define l2n(l,c)        (*((c)++)=(unsigned char)(((l)>>24L)&0xff), \
-                         *((c)++)=(unsigned char)(((l)>>16L)&0xff), \
-                         *((c)++)=(unsigned char)(((l)>> 8L)&0xff), \
-                         *((c)++)=(unsigned char)(((l)     )&0xff))
-
-#undef s2n
-#define s2n(l,c)        (*((c)++)=(unsigned char)(((l)     )&0xff), \
```

Comment:
Amazing. This one is different from all the other `s2n`. Fortunately, idea doesn't use this.